tests/e2e/vsphere_volume_expansion.go

```diff
@@ -2204,3 +2760,55 @@ func sizeInMb(size resource.Quantity) int64 {
 	actualSize = actualSize / (1024 * 1024)
 	return actualSize
 }
+
+func staticProvisioningPreSetUpUtil(ctx context.Context, f *framework.Framework, c clientset.Interface, storagePolicyName string) (*restclient.Config, *storagev1.StorageClass, string) {
+	log := logger.GetLogger(ctx)
+
+	namespace := getNamespaceToRunTests(f)
+	// Get a config to talk to the apiserver
+	k8senv := GetAndExpectStringEnvVar("KUBECONFIG")
+	restConfig, err := clientcmd.BuildConfigFromFlags("", k8senv)
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+	ginkgo.By("Get storage Policy")
+	ginkgo.By(fmt.Sprintf("storagePolicyName: %s", storagePolicyName))
+	profileID := e2eVSphere.GetSpbmPolicyID(storagePolicyName)
+	log.Infof("Profile ID :%s", profileID)
+	scParameters := make(map[string]string)
+	scParameters["storagePolicyID"] = profileID
+	err = c.StorageV1().StorageClasses().Delete(ctx, storagePolicyName, metav1.DeleteOptions{})
+	if !apierrors.IsNotFound(err) {
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+	}
+
+	storageclass, err := createStorageClass(c, scParameters, nil, "", "", true, storagePolicyName)
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+	log.Infof("storageclass name :%s", storageclass.GetName())
+	storageclass, err = c.StorageV1().StorageClasses().Get(ctx, storagePolicyName, metav1.GetOptions{})
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+	log.Infof("storageclass name :%s", storageclass.GetName())
+
+	ginkgo.By("create resource quota")
+	createResourceQuota(c, namespace, rqLimit, storagePolicyName)
+
+	return restConfig, storageclass, profileID
+}
+
+func testCleanUpUtil(ctx context.Context, restClientConfig *rest.Config, c clientset.Interface, cnsRegistervolume *cnsregistervolumev1alpha1.CnsRegisterVolume, namespace string, pvcName string, pvName string) {
+	if guestCluster {
+		c, _ = getSvcClientAndNamespace()
+	}
+	ginkgo.By("Deleting the PV Claim")
+	framework.ExpectNoError(fpv.DeletePersistentVolumeClaim(c, pvcName, namespace), "Failed to delete PVC", pvcName)
+
+	ginkgo.By("Verify PV should be deleted automatically")
+	framework.ExpectNoError(framework.WaitForPersistentVolumeDeleted(c, pvName, poll, supervisorClusterOperationsTimeout))
```

Comment:
`fpv.WaitForPersistentVolumeDeleted` ?

Reply:
Yes, fpv.WaitForPersistentVolumeDeleted   is not present